### PR TITLE
Cl 98682348 implement mobile feed layout change

### PR DESCRIPTION
**Pull request by Wrenderer**
Implement feed design changes based on mockups
 * Implement design change of feed based on mockups - icons to words
 * Modify feeds to only show icons of admins/moderators
 * Feed functionality modifications to match mockup design
    * Add `isAdmin` property to multiple serializer classes; returns if author is admin
    * Merge "add comment" and "view comment" buttons into one.
    * Copy comment form from main page feed to profile page feed where it was missing
    * Modify new comment button/implementation so that one click will both show comments and allow commenting
    * Modify new like button/implementation to be able to show current likes and add a like without multiple tags
 * Add css class `.pipespace` for the separating pipes in feed text
 * Add css class `.feed_action_btn` to re-add pointer cursor to like/comment links in feed
 * Add css class `.post_extra_info` which indents likes, comments, etc. inside feed item
 * Add `.feed-item-displayname` class back to feed item titles
Merge branch 'master' into cl_98682348_implement_mobile_feed_layout_change
 * Add blank href to all non-navigation feed `a` tags
 * Merge master
 * Remove commented out code
 * Add new `is_admin` and `participant` variables to controller specs to get them passing again.

### CODE REVIEW

File: app/assets/stylesheets/social_networking/base.css.scss [hunk 1]
```diff
@@ -138,3 +138,18 @@ div.edit-link {
   top: 2px;
   left: 1px;
 }
+
+/* feed */
```
Comment (michaelwehrley) [inline app/assets/stylesheets/social_networking/base.css.scss]: @Wrenderer is it possible to add bootstrap classes inline instead?

File: app/views/social_networking/homes/show.html.erb [hunk 2]
```diff
@@ -59,65 +59,90 @@
         class="list-group-item"
         id="{{ item.className }}-{{ item.id }}">
       <p>
-        <div ng-if="!item.data.summary.length">
-          <img width="40px;" ng-src="{{ home.profileIconSrcFor(item) }}"> {{ item.summary }}
+        <div class="media feed-item-displayname" ng-if="!item.data.summary.length">
```
Comment (michaelwehrley) [inline app/views/social_networking/homes/show.html.erb]: Is there anywhere in the app that we use `feed_action_btn` or `feed-item-displayname`?

Comment (Wrenderer) [inline app/views/social_networking/homes/show.html.erb]: For future reference we use `feed-item-displayname` in "home_tool_spec.rb" and "profile_spec.rb" of SunnySide.

File: app/views/social_networking/homes/show.html.erb [hunk 3]
```diff
@@ -59,65 +59,90 @@
...
+          <div ng-if="item.isAdmin" class="media-left">
+            <img width="40px" src="{{ home.profileIconSrcFor(item) }}">
```
Comment (ericcf) [inline app/views/social_networking/homes/show.html.erb]: What do you think about adding a CSS class for images with width=40px?

Comment (Wrenderer) [inline app/views/social_networking/homes/show.html.erb]: That sounds fine, any other changes?

Comment (ericcf) [inline app/views/social_networking/homes/show.html.erb]: Nothing that jumped out.

**Comment by ericcf** [comment]
Ok, thanks. Now I'd like to see these commits cleaned up a bit before merging them with master. Take a look at the documentation for git interactive rebase. Thanks!

**Comment by ericcf** [comment]
:+1: :tophat:

**Pull request merged**